packages/form-js-editor/assets/form-js-editor-base.css

```diff
@@ -1,17 +1,19 @@
 .fjs-editor-container {
   --color-children-border: var(--color-grey-225-10-75);
-  --color-children-selected-border: var(--color-blue-205-100-50);
-  --color-children-selected-background: var(--color-blue-205-100-95);
-  --color-children-hover-border: var(--color-blue-205-100-50);
+  --color-children-selected-border: hsl(219, 99%, 53%);
```

Comment:
Should we tidy these up?

Reply:
I'll look whether I find a good form-js representation of this, will anyway be overridden via #557 rather soon.